cfme/tests/services/test_rest_services.py

```diff
@@ -486,6 +487,8 @@ def test_delete_orchestration_templates(self, rest_api, orchestration_templates,
         collection = rest_api.collections.orchestration_templates
         if from_detail:
             methods = ['post', 'delete']
+            if BZ('1414881', forced_streams=['5.7', 'upstream']).blocks:
+                methods[0] = 'delete'  # because the 'post' method fails
```

Comment:
Personally I'd rather see post removed from methods and a TODO to add it back when the bug is resolved.